app/frontend/package.json

```diff
@@ -20,9 +20,14 @@
     "@react-spring/web": "^9.7.3",
     "marked": "^13.0.0",
     "dompurify": "^3.0.6",
+    "i18next": "^23.11.5",
+    "i18next-browser-languagedetector": "^8.0.0",
+    "i18next-http-backend": "^2.5.2",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "react-router-dom": "^6.23.1",
+    "react-helmet": "^6.1.0",
```

Comment:
The react-helmet is no longer maintained. The creator seems to recommend react-helmet-async instead, which gets a much better score on Snyk (https://snyk.io/advisor/npm-package/react-helmet-async)

Reply:
Thank you, changed to react-helmet-async.